### PR TITLE
Add repository condition to stale workflow

### PR DESCRIPTION
**Pull request by nielsvanvelzen**
**Changes**
- Add repository condition to stale workflow
  - Now disabled for forks
- Change runs-on for gradle workflows (accidently used ubuntu-latest)
- Change cron schedule to trigger 30 minutes before the gradlew-update workflow
  - If both workflows trigger GitHub notifications the gradle update notification will be on the top

**Issues**

### CODE REVIEW

File: .github/workflows/gradlew-validate.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   validate:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.4
```
Comment (Maxr1998) [inline .github/workflows/gradlew-validate.yaml:13]:
```suggestion
    runs-on: ubuntu-20.04
```

File: .github/workflows/gradlew-update.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   update:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.4
```
Comment (Maxr1998) [inline .github/workflows/gradlew-update.yaml:9]:
```suggestion
    runs-on: ubuntu-20.04
```

**Review by Maxr1998 — CHANGES_REQUESTED** [review]

File: .github/workflows/repo-stale.yaml [hunk 3]
```diff
@@ -12,6 +12,7 @@ permissions:
 jobs:
   stale:
     runs-on: ubuntu-20.04
+    if: github.repository == 'jellyfin/jellyfin-android'
```
Comment (h1dden-da3m0n) [inline .github/workflows/repo-stale.yaml:15]: sorry for forgetting that one :man_facepalming:

**Review by h1dden-da3m0n — APPROVED** [review]
as soon as Max's comments are in this will not only work but LGTM

**Review by Maxr1998 — APPROVED** [review]

**Pull request merged**